tensorboard/components/vz_example_viewer/vz-example-viewer.ts

```diff
@@ -371,21 +372,19 @@ Polymer({
   },
 
   haveSaliency: function() {
+    // Saliency-coloring waits until the display elements have been updated
+    // to avoid coloring divs that are then re-ordered/re-used/re-named by
+    // the dom-repeat of feature divs.
+    this.debounce('haveSaliency', () => this.haveSaliencyImpl(),
+                  ATTRIBUTION_TIMER_DELAY_MS);
+  },
+
+  haveSaliencyImpl: function() {
```

Comment:
May want to prefix it by "_" to denote privateness.

Reply:
done